src/khoj/interface/web/chat.html

```diff
@@ -1988,6 +2230,14 @@
             margin: 10px;
         }
 
+        li.fileName {
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            max-width: 250px;
+            background-color:var(--primary);
```

Comment:
Some other styling that may potentially be useful. Right now, the file names are bunched up a bit too close to one another. This should help break it up a bit:
```
            border-radius: 10px;
            margin-bottom: 4px;
            margin-top: 4px;
            padding-top: 4px;
            padding-bottom: 4px;
            border: 1px solid var(--primary);
```